README.md

```diff
@@ -95,10 +95,6 @@ This is a modification of the `ec2_node` community handler. It caches the list
 of ec2 instances from the Amazon API and will automatically remove servers
 from Sensu if they do not exist in the API.
 
-#### graphite
-
-Standard handler, sends graphite metrics.
```

Comment:
Yea, the fact that it is a standard handler, to me, hints that it shouldn't really be in this puppet module. This module isn't meant to store and deploy every conceivable sensu handler, just our special ones.